File: docs/snippets/all/archetypes/ellipses2d_simple.py
"""Log a simple 2D ellipse."""

import simplant_lab as rr

rr.init("rerun_example_ellipses2d", spawn=True)

rr.log("simple", rr.Ellipses2D(half_sizes=[(2.0, 1.0)], centers=[(0.0, 0.0)]))

File: docs/snippets/all/archetypes/ellipses2d_simple.rs
//! Log some very simple 2D ellipses.

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let rec =
        simplant_lab::RecordingStreamBuilder::new("rerun_example_ellipses2d")
            .spawn()?;

    rec.log(
        "simple",
        &simplant_lab::Ellipses2D::from_centers_and_half_sizes(
            [(0.0, 0.0)],
            [(2.0, 1.0)],
        ),
    )?;

    Ok(())
}
